lib/assessment.js

```diff
@@ -29,7 +32,10 @@ var newArray = function newArray(length, defaultValue) {
 // invoking the argument defaultsFunction
 // defaultsFunction should be passed the index of the array element to be set
 var newArray2 = function newArray2(length, defaultsFunction) {
-
+  var defaultsFunction = function (){
+    ///??????? I DON'T KNOW!!!!!!!!!
```

Comment:
This one is different from the previous question in step 2:

2. Loop from 0 to `length` and store the return value of `defaultsFunction` invoked on `i` in `array[i]`.